src/imports.rs

```diff
@@ -39,10 +40,10 @@ impl Rewrite for ast::ViewPath {
                 let path_str = try_opt!(path.rewrite(context, width - ident_str.len() - 4, offset));
 
                 Some(if path.segments.last().unwrap().identifier == ident {
-                         path_str
-                     } else {
-                         format!("{} as {}", path_str, ident_str)
-                     })
+                        path_str
```

Comment:
Yeah, it is aligning to 5 tabs rather than using visual indenting. I think I prefer the previous formatting, but it's not a bug with the current PR, it's just that the current PR means that this expression is getting formatted at all.